test/acceptance/features/bindAppToServiceWithLabelSelector.feature

```diff
@@ -0,0 +1,111 @@
+Feature: Bind an application with label selector to a service
+
+    As a user of Service Binding Operator
+    I want to bind applications with label selector to services it depends on
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+
+    Scenario: Bind an application with label selector to backend service
```

Comment:
application resource has labels attached, but label selector is used inside service binding to define what application should be bound. Could we be a bit more verbose about the setup and the expected outcome? For example:

```
Bind successfully an application owning a given label to a service
```

Reply:
Done. af6e990baa021c05e9fba2e7cb713ac745500ba9